client/js/game.js

```diff
@@ -12712,7 +12712,14 @@ function Game() {
                         for (var j=0;j<13;++j) {
                             if (num<64) {
                                 if (CQW.flash.history[showFlash].players[num].name!==kongregate.services.getUsername()) {
-                                    text(ctx,(num+1)+"."+CQW.flash.history[showFlash].players[num].name,W*0.345+(W*0.11*i),H*0.19+(30*j),"28px"+FONT,"white","center","middle");
+									var fcolor='white';
+									if(CQW.flash.history[showFlash].players[num].res!==undefined) {
+										if(CQW.flash.history[showFlash].players[num].res.w.indexOf(CQW.flash.history[showFlash].players[num].id))
+											fcolor='darkgreen';
```

Comment:
I agree that green seems more intuitive, but blue is less likely to cause problems for colorblind people.
```suggestion
											fcolor='darkblue';
```